src/Restaurant.js

```diff
@@ -1,17 +1,19 @@
-import React from "react";
+import React, { Component } from 'react';
+import Orders from './Orders' 
 
-const Restaurant = () => {
-  const pizzas = 0;
-  return (
-    <div>
-      <h3>Restaurant Orders</h3>
-      <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
-      </ul>
-    </div>
-  );
-};
+class Restaurant extends Component {
+  render() {
+    const pizzas = 0;
+    return (
+      <div>
+        <h3>Restaurant Orders</h3>
+        <ul>
+          <Orders orderType="Pizzas" />
+          <Orders orderType="Salads" />
```

Comment:
Can you see how making our component reusable (but configurable via props) is really useful?